src/gt4py/cartesian/gtc/numpy/oir_to_npir.py

```diff
@@ -208,8 +216,8 @@ def visit_VerticalLoopSection(
     ) -> npir.VerticalPass:
         return npir.VerticalPass(
             body=self.visit(node.horizontal_executions, **kwargs),
-            lower=node.interval.start,
-            upper=node.interval.end,
+            lower=self.visit(node.interval.start),
+            upper=self.visit(node.interval.end),
```

Comment:
I think it would be nice to pass the `kwargs` on in case we need them when we actually implement `RuntimeAxisBounds` in the numpy backend. PR https://github.com/GridTools/gt4py/pull/2357 made me sensitive to silently dropping `kwargs`. Maybe I'm overly cautions now ... 

```suggestion
            lower=self.visit(node.interval.start, **kwargs),
            upper=self.visit(node.interval.end, **kwargs),
```